spec/features/state_file/complete_intake_spec.rb

```diff
@@ -245,6 +245,19 @@
       expect(page).to have_text I18n.t('state_file.questions.az_public_school_contributions.index.lets_review')
       click_on I18n.t("general.continue")
 
+      expect(page).to have_text I18n.t('state_file.questions.az_qualifying_organization_contributions.form.main_heading', filing_year: Rails.configuration.statefile_current_tax_year)
+      choose I18n.t("general.affirmative")
+      fill_in "az321_contribution_charity_name", with: "Center for Ants"
```

Comment:
did you see the factory or did we just happen to come up with the same charity name